__tests__/place.test.ts

```diff
@@ -42,26 +42,42 @@ const mockUpdateOne = jest
 			mockCollection.pop();
 			mockCollection.push({
 				_id: id,
-				avgBraille:
-					newPlace.avgBraille !== undefined
-						? newPlace.avgBraille
-						: old?.avgBraille || null,
-				avgFontReadability:
-					newPlace.avgFontReadability !== undefined
-						? newPlace.avgFontReadability
-						: old?.avgFontReadability || null,
-				avgStaffHelpfulness:
-					newPlace.avgStaffHelpfulness !== undefined
-						? newPlace.avgStaffHelpfulness
-						: old?.avgStaffHelpfulness || null,
-				avgNavigability:
-					newPlace.avgNavigability !== undefined
-						? newPlace.avgNavigability
-						: old?.avgNavigability || null,
-				avgGuideDogFriendly:
-					newPlace.avgGuideDogFriendly !== undefined
-						? newPlace.avgGuideDogFriendly
-						: old?.avgGuideDogFriendly || null,
+				guideDogAvg:
+					newPlace.guideDogAvg !== undefined
+						? newPlace.guideDogAvg
+						: old?.guideDogAvg || null,
```

Comment:
A little weird to use a ternary and then an OR operator. Either a double ternary or (preferred:) double OR would prob make more sense for all of these
`newPlace?.guideDogAvg || old?.guideDogAvg || null`